src/key.rs

```diff
@@ -1321,7 +1321,11 @@ impl fmt::Display for InvalidParityValue {
 
 #[cfg(feature = "std")]
 #[cfg_attr(docsrs, doc(cfg(feature = "std")))]
-impl std::error::Error for InvalidParityValue {}
+impl std::error::Error for InvalidParityValue {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        None
```

Comment:
Is the point of this to express "no, we didn't forget about this"?

Reply:
I'm going to remove this, I did not think thoroughly enough when doing this to realise I was just adding the default implementation :) Thanks